pkg/log/log.go

```diff
@@ -0,0 +1,34 @@
+package log
+
+import (
+	"fmt"
+	"io"
+)
+
+type StderrLogger struct {
+	Stderr io.Writer
+}
+
+func (l StderrLogger) Actionf(format string, a ...interface{}) {
+	fmt.Fprintln(l.Stderr, `►`, fmt.Sprintf(format, a...))
+}
+
+func (l StderrLogger) Generatef(format string, a ...interface{}) {
```

Comment:
what does `Generate` mean in the context of logging :thinking: the others I can get onboard with, athough `Action` is also a bit ambiguous. I can't really tell from the name of either of them when they should be used. 

I know we just copied from flux but we can always adjust it :smile:

Reply:
good point! we only use `Generate` once so it could be replaced by something else, like `Action`. `Action` is used for logs that just announce what's going to happen next but it's very similar to `Waiting`